cli/src/commands/jobs.rs

```diff
@@ -158,12 +161,13 @@ pub async fn handle_submission(api: &mut PhylumApi, matches: &clap::ArgMatches)
                     }
                     let pkg_version = pkg_info.pop().unwrap();
                     let pkg_name = pkg_info.join(":");
+                    let pkg_type = purl::PackageType::from(request_type);
+                    let purl = Purl::builder(pkg_type, pkg_name)
+                        .with_version(pkg_version)
+                        .build()?
+                        .to_string();
 
-                    packages.push(PackageDescriptor {
-                        name: pkg_name.to_owned(),
-                        version: pkg_version.to_owned(),
-                        package_type: request_type.to_owned(),
-                    });
+                    packages.push(ParsedPurl { purl, lockfile_path: None });
```

Comment:
Guess the path to file used for batch submissions can be included as a lock file path?